### PR TITLE
Create image from icon theme and adding a background to layout

### PR DESCRIPTION
**Pull request by anaselli**
This PR just fixes issue #86 and #87

### CODE REVIEW

File: src/YGImage.cc [hunk 1]
```diff
@@ -10,6 +10,7 @@
 #include "YImage.h"
 #include "ygtkimage.h"
 #include <string.h>
+#include <boost/filesystem.hpp>
 
 static inline bool endsWith (const std::string &str1, const char *str2)
```
Comment (shundhammer) [inline src/YGImage.cc:15]: JFYI: There is also `boost::algorithm::ends_with()` which might take some of the maintenance burden off your shoulders in the future.

https://techoverflow.net/2022/01/25/how-to-use-boost-string-algorithm-ends_with-a-minimal-example/

But it's your choice whether or not you want to use that.

**Review by shundhammer — APPROVED** [review]
LGTM

(again, I don't know much about Gtk)

**Comment by shundhammer** [comment]
I was wondering if we need to give you permission to merge to that repository, and then I found that I don't seem to have permission to grant anybody permission. ;-)

**Comment by anaselli** [comment]
:) i can merge, but I need at least one reviewer, don't know if it is why i am the committer or just because one of you is needed...

**Pull request merged**